docs/changelog.rst

```diff
@@ -2,6 +2,31 @@
 
 Changelog
 =========
+0.11.0 (2021-07-28)
+------------------
+- CausalML surpassed `2K stars <https://github.com/uber/causalml/stargazers>`_!
+- We have 3 new community contributors, Jannik (`@jroessler <https://github.com/jroessler>`_), Mohamed (`@ibraaaa <https://github.com/ibraaaa>`_), and Leo (`@lleiou <https://github.com/lleiou>`_). Thanks for the contribution!
+
+Major Updates
+~~~~~~~~~~~~~
+- Make tensorflow dependency optional by @jeongyoonlee (`#343 <https://github.com/uber/causalml/pull/343>`_)
+- Add delta-delta-p (ddp) tree inference approach by @jroessler (`#327 <https://github.com/uber/causalml/pull/327>`_)
+- Add conda env files for Python 3.6, 3.7, and 3.8 by @jeongyoonlee (`#324 <https://github.com/uber/causalml/pull/324>`_)
```

Comment:
and Python 3.9 :)

Reply:
Thanks! I added that in the first #343 description - does this look good?